lib/problem_check/twitter_login.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ProblemCheck::TwitterLogin < ProblemCheck
+  self.perform_every = 24.hours
+
+  def call
+    return [] if !authenticator.enabled?
+    return [] if authenticator.healthy?
+
+    [Problem.new(I18n.t("dashboard.twitter_login_warning"), priority: "high", identifier:)]
+  end
+
+  private
+
+  def authenticator
+    Auth::TwitterAuthenticator.new
```

Comment:
maybe we should memoize this? not a big deal really just a thought

Reply:
Might as well.